libs/model/src/policies/ChainEventCreated.policy.ts

```diff
@@ -5,12 +5,29 @@ import { handleCommunityStakeTrades } from './handlers/handleCommunityStakeTrade
 import { handleNamespaceDeployed } from './handlers/handleNamespaceDeployed';
 import { handleNamespaceDeployedWithReferral } from './handlers/handleNamespaceDeployedWithReferral';
 import { handleReferralFeeDistributed } from './handlers/handleReferralFeeDistributed';
+import {
+  handleTokenDelegated,
+  handleTokenLockDurationIncreased,
+  handleTokenLocked,
+  handleTokenMerged,
+  handleTokenPermanentConverted,
+  handleTokenUndelegated,
+  handleTokenUnlocked,
+} from './handlers/handleTokenStaking';
 
 const chainEventInputs = {
   CommunityStakeTrade: events.CommunityStakeTrade,
   NamespaceDeployed: events.NamespaceDeployed,
   NamespaceDeployedWithReferral: events.NamespaceDeployedWithReferral,
   ReferralFeeDistributed: events.ReferralFeeDistributed,
+  // TokenStaking
```

Comment:
Is there any reason we don't make this its own Policy?

Reply:
This is part of the chain events ACL layer... will adjust to latest convention in future PRs if necessary. At this point what we need is to get those events in the maps so we can continue working downstream.

Reply:
ChainEvent Policy is not the ACL. With the current set-up none of these events will be processed into the created handlers because these events don't have any mappers. You must create mappers [here](https://github.com/hicommonwealth/commonwealth/blob/master/libs/model/src/services/evmChainEvents/chain-event-utils.ts) or these events will simply be ignored by EVM CE.